doc/themes/the-things-stack/layouts/shortcodes/ttnv2.html

```diff
@@ -0,0 +1 @@
+The Things Network Stack V2
```

Comment:
cc: @benolayinka for this one.

Reply:
I moved it because it seems like a basic element of TTS that doesn't depend on anything site specific, and it could migrate cleanly to another site as part of the theme.